Server/src/main/java/app/ModelController.java

```diff
@@ -143,6 +143,7 @@ private void trainModel(TrainedModel trainedModel) throws Exception {
         ModelStrategy modelStrategy = getModelStrategy(trainedModel);
         if (modelStrategy != null) {
             ModelsController modelsController = new ModelsController(modelStrategy);
+            trainedModel.setModelStrategy(modelStrategy);
```

Comment:
isn't that redundant since you are already getting the model Strategy of the trained model from line 143? It feels redundant.

Reply:
Actually the modelStrategy is in the trainedmodel so we're both wrong, i adjusted the code. The get ModelStrategy is no longer used but i have a feeling we will need it in the future so i'm not gonna remove it right now